CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+* **colors:** Updated colors and gradients. Removes Mozilla brand colors, magenta, teal. Adds pink and violet hues. Adds a lighter shade for each color. (#30)
```

Comment:
Nit: suggest adding `(breaking)` to the beginning of this description. We've started doing this in Protocol to more easily identify breaking changes. This is particularly helpful when deciding what the next NPM version number should be.